decidim-core/app/packs/stylesheets/decidim/_header.scss

```diff
@@ -24,10 +24,6 @@ header {
     @apply mt-28 md:mt-0;
   }
 
-  #menu-bar-container {
-    @apply mt-24 md:mt-0;
```

Comment:
This fixes a bug where there's an unnecessary margin-top. I only can reproduce it in nightly :/ 

![Screenshot of the margin-top in the header bug](https://github.com/decidim/decidim/assets/717367/87b59c12-e1ac-4f33-b639-1099db52e41a)

It doesn't seem to break anything so I think it's OK to merge this

Reply:
I am against this change, within this PR. 
The empty space that you see on nightly is caused by omnipresent banner. Removing the class above, without having omnipresent banner enabled, will cause other pages to missbehave: 

This is how the show proposal page looks like.
![image](https://github.com/decidim/decidim/assets/105683/c3e5c442-337c-4cc5-a6f1-e82b962afbf8)

Reply:
when omnipresent banner is displayed, and when the admin bar is being shown, the following content is rendered 
![image](https://github.com/decidim/decidim/assets/105683/7fbd431d-fe92-44b0-8516-b262c2ffb44c)

Adding some CSS styling to omnipresent-banner, we can see the following result:
```
  z-index: 50;
  position: fixed;
``` 
![image](https://github.com/decidim/decidim/assets/105683/5d8fb91d-c9be-4089-983f-0f702f6fdd8a)

Reply:
Oh that's why I couldn't reproduce it locally! I answered to this concern at https://github.com/decidim/decidim/pull/13025#issuecomment-2213091167